src/components/HorizontalList/HorizontalListStaffing/HorizontalListStaffing.tsx

```diff
@@ -0,0 +1,140 @@
+import { HorizontalList } from "../HorizontalList";
+import { useEffect, useState } from "react";
+export function HorizontalListStaffing() {
```

Comment:
Let's make this component more generic by not calling it `HorizontalListStaffing`, but rather something like `HorizontalListBackground`.